.github/workflows/tools.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/validators/instance_validator
     - name: Instance Yaml Validator Tests
-      if: steps.changes.outputs.instance_validator == 'true'
+      if: steps.changes.outputs.instance_validator == 'true' OR steps.changes.outputs.type_validator == 'true'
```

Comment:
the or is not supported, please use the expression here: https://docs.github.com/en/actions/learn-github-actions/expressions#operators

Reply:
Done.